### DISCUSSION TITLE
DB2 for i - Closing file opened with SQL

### DISCUSSION THREAD

**Discussion by JonasEngstedt** [codefori/vscode-ibmi#1795] [Q&A] [answered]
Hi there!

This might be a stupid question but I've been searching far and wide through VSCode and can't find any options to actually close a result set in db2 for i. Closest thing I could find was the keyboard shortcut for : "command": "vscode-db2i.resultset.removeView". But that doesn't actually close the file when looking at the job. Does anyone have any ideas? Sorry if it's blatantly obvious and I'm just being stupid :)

**Comment by chrjorgensen** [comment]
@JonasEngstedt 

Hi Jonas.

I think currently the only way to close a result set is to end the server job seen in `SQL JOB MANAGER`.
Have you tried this?

**Reply by JonasEngstedt** [reply]
Hi, yes I've tried that. Thought it was a bit clumsy though. I was looking for something more like ACS solution by just closing the result set at the press of a button. But I might have to settle for that then. Thanks.

**Reply by JonasEngstedt** [reply]
Funnily enough, closing the job doesn't actually close the result set :D

**Comment by worksofliam** [comment, marked as answer]
> can't find any options to actually close a result set in db2 for i

That would be because there isn't a way to do that! The result set will just clear when you run a new statement again.

**Reply by JonasEngstedt** [reply]
Alrighty then. Thanks!
